plugins/slack/server/auth/slack.ts

```diff
@@ -125,27 +128,36 @@ if (env.SLACK_CLIENT_ID && env.SLACK_CLIENT_SECRET) {
   passport.use(strategy);
 
   router.get("slack", passport.authenticate(providerName));
-
   router.get("slack.callback", passportMiddleware(providerName));
 
   router.get(
-    "slack.commands",
+    "slack.post",
```

Comment:
Centralized the three callbacks for Slack under a single root, as adding additional redirect-uris for new functionality is particularly painful with Slack.